src/Str3.java

```diff
@@ -53,4 +53,4 @@ public int countYZ (String str) {
     }
 
 
-}
+} 
```

Comment:
Can't find String-1 tasks here. It's good that you've managed to solve more complicated tasks, but you were supposed to implement 10 tasks of String-1 as well.